src/components/engagement/engagement.rules.ts

```diff
@@ -20,6 +20,7 @@ import {
 
 interface Transition extends EngagementStatusTransition {
   projectStepRequirements?: ProjectStep[];
+  inChangeset?: boolean;
```

Comment:
Add `inChangeset` prop and it will have three states now.

- `true`: transition will be available only in changeset mode
-  `false`: transition will be available not in changeset mode but in the normal workflow
-  `undefined`: transition will be available for both workflows.